service/src/main/java/bio/terra/profile/service/profile/exception/MissingRequiredProvidersException.java

```diff
@@ -0,0 +1,11 @@
+package bio.terra.profile.service.profile.exception;
+
+import bio.terra.common.exception.ErrorReportException;
+import java.util.List;
+import org.springframework.http.HttpStatus;
+
+public class MissingRequiredProvidersException extends ErrorReportException {
```

Comment:
Should you update the Swagger to document this possible exception from the create method?

Reply:
Done, see https://github.com/DataBiosphere/terra-billing-profile-manager/blob/45e2e4491708777d063fdbab60459b013909869d/service/src/main/resources/api/service_openapi.yaml#L65-L71